setup.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+import os
+
+from setuptools import find_packages, setup
+
+
+here = os.path.dirname(__file__)
+
+with open(os.path.join(here, 'VERSION')) as f:
+    version = f.read().strip()
+
+with open(os.path.join(here, 'requirements.txt')) as f:
+    install_requires = f.read().strip().split('\n')
+
+
+setup(
+    name='clouseau',
+    version=version,
+    description='Tool to find out some clues after crashes in using data from Socorro, Bugzilla and mercurial',
```

Comment:
We should improve the description both here and on GitHub, since the project evolved to be more than this.

We should also decide whether we want to keep the `clouseau` name for the library, or use a different name for the library and keep `clouseau` for the tool.

See also #51.

@calixteman @sylvestre what do you think?